hs_core/templates/pages/genericresource.html

```diff
@@ -850,6 +853,28 @@ <h4 class="modal-title" id="myModalLabel">Add file to resource</h4>
                                 $res_title.text(updated_title);
                             }
                         }
+
+                        if (json_response.hasOwnProperty('metadata_status')){
+                            if(json_response.metadata_status !== $('#metadata-status').text()){
+                                $('#metadata-status').text(json_response.metadata_status);
+                                if(json_response.metadata_status == "Sufficient to make public")
+                                {
+                                    alert("Metadata Status:Sufficient to make public");
```

Comment:
Oh, my mistake. Well, the problem is that alerts() are blocking. They block the browser UI, in some browsers even for other tabs and windows, until the user locates and closes the alert dialog.

A common replacement is to have a hidden or empty DIV somewhere on the page, styled as a message and waiting for you to call text() on it to inject the message and reveal the message box as part of the page itself.